client/components/PubShareDialog/PubShareDialog.js

```diff
@@ -92,7 +92,7 @@ const MembersOptions = (props) => {
 	const localMembers = membersByType[activeTargetType];
 
 	return (
-		<>
+		<React.Fragment>
```

Comment:
Matter of taste, but we can use the `<> </>` syntax for these now, which feels cleaner to me.

Reply:
I've had 3 (though now 2) reasons for preferring `<React.Fragment>`:

1. The sublime text babel syntax highligher would fail on `<>` syntax causing the whole file's syntax highlighting to break. Digging into it, I [found an issue](https://github.com/babel/babel-sublime/issues/368) pointing towards a pre-release that fixes this. I've installed that pre-release, so the syntax highlighting isn't an issue for me anymore.
2. I prefer the explicitness of declaring `React.Fragment` because it feels friendlier to folks unfamiliar with JSX or Fragments. If you come across that and don't know what a Fragment is, it's clear that 1) it's part of React, and 2) you could search `React.Fragment` in a search engine to learn more. If you come across `<>` without knowing what it is, it's not immediately clear whether that's part of React, HTML, or some webpack pre-processor. A understandably incorrect search "html empty tag" in google doesn't land you near React Fragments.
3. `<>` doesn't support keys whereas `<React.Fragment>` does. [React docs also hint](https://reactjs.org/docs/fragments.html#keyed-fragments) that future attributes may be supported, which would only be available on the `<React.Fragment>` flavor. My overarching preference is to consistently use one syntax or the other, and given that we already have a few cases where we use `<React.Fragment key={something}>`, I lean towards consistently using that syntax.

Reply:
Thanks, some good food for thought here!